src/main/java/com/exasol/adapter/dynamodb/documentpath/ObjectLookupPathSegment.java

```diff
@@ -0,0 +1,31 @@
+package com.exasol.adapter.dynamodb.documentpath;
+
+/**
+ * Path segment defining an key lookup of an object e.g object.property
```

Comment:
```suggestion
 * Path segment defining a key lookup of an object, e.g. object.property.
```

Reply:
Short remark on `e.g.` and `e.i.`, I remember that there is always comma before them.